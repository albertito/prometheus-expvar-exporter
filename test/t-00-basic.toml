listen_addr = ":30080"

[target]
url = "http://localhost:30081/t-00-basic.vars"
