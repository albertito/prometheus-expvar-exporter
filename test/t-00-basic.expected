# HELP a_float expvar "a-float"
# TYPE a_float untyped
a_float 1.2
# HELP an_int expvar "an-int"
# TYPE an_int untyped
an_int 26
# HELP with_unsupported_characters expvar "with/unsupported-characters"
# TYPE with_unsupported_characters untyped
with_unsupported_characters 55
